### ISSUE TITLE
native library `kernel32` is linked by more than one package, and con only be linked to by one package

### ISSUE THREAD

**Issue by NobbZ** [PistonDevelopers/piston#1004]
When trying to follow the tutorial (rotating rectangle) I get the following errormessage directly after downloading of `khronos_api` happened:

```plain
native library `kernel32` is being linked to by more than one package, and can only be linked to by one package

  kernel32-sys v0.1.4
  kernel32-sys v0.2.0
```

Tried with rust-channels `stable` and `nightly`, using `multirust-rs` to manage them.

I'm on Gnome-Ubuntu 14.04 and having all external dependencies installed that are mentioned in the tutorial (https://github.com/PistonDevelopers/Piston-Tutorials/tree/master/getting-started), namely `libfreetype6-dev`.

**Comment by mitchmindtree** [comment]
Please see these related issues:
https://github.com/PistonDevelopers/conrod/issues/622
https://github.com/retep998/winapi-rs/issues/238

We should be able to fix this once [this](https://github.com/tomaka/glutin/pull/652) gets merged.

Sorry for the trouble!

**Comment by NobbZ** [comment]
I have to be sorry for the noise. I did a search but neither google nor github gave me something related :(

**Comment by retep998** [comment]
This should have been fixed by me publishing `0.2.1` of `kernel32-sys`.

**Comment by NobbZ** [comment]
OK, seems to be solved now. Thank you all!
